src/ui/components/BuildResponse/index.tsx

```diff
@@ -60,40 +63,32 @@ const BuildResponse: FunctionComponent<BuildResponseProps> = memo(
               )}
             </AlertTitle>
             <p>
-              An error has occured, see the above log for the exact error
-              message. If you have not already done so, visit{' '}
-              <DocumentationLink
-                firmwareVersion={firmwareVersionData}
-                url="https://www.expresslrs.org/"
-              >
-                Expresslrs.org
-              </DocumentationLink>{' '}
-              and read the{' '}
-              <DocumentationLink
-                firmwareVersion={firmwareVersionData}
-                url="https://www.expresslrs.org/quick-start/getting-started/"
-              >
-                Flashing Guide
-              </DocumentationLink>{' '}
-              for your particular device as well as the{' '}
-              <DocumentationLink
-                firmwareVersion={firmwareVersionData}
-                url="https://www.expresslrs.org/quick-start/troubleshooting/#flashingupdating"
-              >
-                Troubleshooting Guide
-              </DocumentationLink>
-              . If you are still having issues after reviewing the
-              documentation, please copy the build logs above to an online paste
-              site and post in the #help-and-support channel on the{' '}
-              <DocumentationLink
-                firmwareVersion={firmwareVersionData}
-                url="https://discord.gg/dS6ReFY"
-              >
-                ExpressLRS Discord
-              </DocumentationLink>{' '}
-              with a link to the logs and other relevant information like your
-              device, which flashing method you were using, and what steps you
-              have already taken to resolve the issue.
+              <Trans
+                i18nKey="BuildResponse.ErrorDetails"
+                defaults="An error has occured, see the above log for the exact error message. If you have not 
```

Comment:
Yeah, I know. I thought it would be friendly for developers if we leave the actual text there.
I will remove it if it is unnecessary.

Reply:
While I do see the thought behind it being helpful, the text in the Trans component can be complex, but since most of the translations are using the useTranslation hook and only supplying the key and not the text itself, only having the key for the Trans component would keep things consistent.  Also, since that text isn't actually used, it is just there for reference, it is likely to get out of sync with the translation file if is forgotten that both need to be updated.  At that point, the additional text that was supposed to be helpful can actually cause confusion since it says one thing, but what is actually appearing on the screen is something different.

Reply:
Fixed